app/models/concerns/demodulizes_policy_class.rb

```diff
@@ -0,0 +1,9 @@
+module DemodulizesPolicyClass
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def policy_class
+      "#{name.demodulize}Policy"
+    end
+  end
+end
```

Comment:
This is the logic to demodulize all of the model policies.

This could likely be improve to retain non-service namespaces.

```ruby
def policy_class
  return "#{name.delete_prefix("Claims::")}Policy" if name.starts_with("Claims::")
  return "#{name.delete_prefix("Placements::")}Policy" if name.starts_with("Placements::")
end
```

This would allow automatic lookup for policies of variants of models, like `School::Location` vs `Provider::Location`, if the need arised.